### PR TITLE
Toolbox release workflow

### PR DESCRIPTION
**Pull request by tbwiss**
# Description

https://graphql-toolbox.neo4j.io is pointing to a AWS S3 bucket where the GraphQL Toolbox web assets are stored. We need a GHA release workflow to push the latest changes to this already existing AWS S3 bucket and do the usual version updates etc.
With this release workflow we will NOT publish anything to npm.

TODO:
- creating a "build user" on AWS for the `AWS_ACCESS_KEY` (should be available 17th of May)

### CODE REVIEW

**Comment by neo4j-team-graphql** [comment]
Looks like you've updated the UI!

Check out your changes at
 http://neo4j-graphql-toolbox-stage-1426.surge.sh (=> for local development; use for instance [Neo4j Desktop](https://neo4j.com/download/))
or
 https://neo4j-graphql-toolbox-stage-1426.surge.sh (=> use to connect to an [Aura Free](https://neo4j.com/cloud/aura-free/) dbms)

File: .github/workflows/release-toolbox.yml [hunk 1]
```diff
@@ -0,0 +1,204 @@
...
+      - name: Extract prerelease from version
+        id: get-prerelease
+        run: |
+          prerelease=$(echo ${{ needs.bump-version.outputs.version }} | cut -d@ -f3 | cut -d- -f2)
+          echo "::set-output name=prerelease::$prerelease"
+      - name: Build Changelog
+        id: github-release
+        uses: mikepenz/release-changelog-builder-action@v3
+        with:
+          configuration: "config/release-changelog-builder-action/configuration-graphql-toolbox.json"
```
Comment (darrellwarde) [inline .github/workflows/release-toolbox.yml:191]: This file will need adding, can be copied and slightly tweaked from one of the others. 🙂

**Review by darrellwarde — CHANGES_REQUESTED** [review]
Looks good, just need the one config file adding from what I can see!

File: .github/workflows/release-toolbox.yml (hunk 1 above)
Comment (tbwiss) [reply .github/workflows/release-toolbox.yml:191]: Thanks for pointing it out. I will do that!

**Review by darrellwarde — APPROVED** [review]
This looks like it should be good to me!

**Pull request merged**